torchmultimodal/modules/layers/codebook.py

```diff
@@ -172,34 +212,44 @@ def _quantize(self, encoded_flat: Tensor) -> Tuple[Tensor, Tensor]:
         distances = torch.cdist(encoded_flat, self.embedding, p=2.0) ** 2
 
         # Encoding - select closest embedding vectors
-        codebook_indices = torch.argmin(distances, dim=1)
+        codebook_indices_flat = torch.argmin(distances, dim=1)
 
         # Quantize
-        quantized_flat = self.embedding[codebook_indices]
+        quantized_flat = self.embedding[codebook_indices_flat]
```

Comment:
replace this with lookup?

Reply:
I replaced with `F.embedding` as it's the same effect and only 1 line of code.